docs/releases/1.11.txt

```diff
@@ -726,6 +726,8 @@ Miscellaneous
 
 * Support for IPython < 1.0 is removed from the ``shell`` command.
 
+* Support for ``psycopg2`` < 2.5.4 is removed.
```

Comment:
In 1.8, we put a similar note under "Support for PostgreSQL versions older than 9.0", so you could put this in the similar "Dropped support for PostgreSQL 9.2 and PostGIS 2.0" section. I would say, "Also, the minimum supported version of psycopg2 is increased from 2.4.5 to 2.5.4."